packages/rest/src/lib/REST.ts

```diff
@@ -17,6 +17,17 @@ export interface RESTOptions {
 	 * @default 'https://cdn.discordapp.com'
 	 */
 	cdn: string;
+	/**
+	 * The number of invalid REST requests (those that return 401, 403, or 429) in a 10 minute window between emitted warnings (0 for no warnings).
```

Comment:
10 minute window?

Reply:
That's what the original did, its arbitrary as far as I can tell.